CONTRIBUTING.md

```diff
@@ -26,9 +26,70 @@ If you find something you'd like added, feel free to
 sure to [review existing issues](https://github.com/workday/canvas-tokens/issues) first to
 reduce duplicates.
 
-### Submitting a Pull Request
+### Making A Contribution
 
-### Git Guidelines
+#### Forking the Repo
 
-This project uses [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) to keep
-commit messages consistent.
+Once you have an issue, you're ready to start working your contribution! If this is your first contribution, you'll need to fork this repo. You can either do this through GitHub's UI or the [`gh` CLI](https://cli.github.com/manual/gh_repo_fork).
+
+```sh
+gh fork repo Workday/canvas-tokens
+```
+
+Once forked, you can clone it locally, and set upstream and set your remotes
+
+```sh
+# clone the repo
+gh repo clone Workday/canvas-tokens
+# change the directory
+cd canvas-tokens
+# rename the origin remote to upstream
+git remote rename origin upstream
+# set the origin remote to your fork
+git remote add origin git@github.com:[your-username]/canvas-tokens.git
+```
+
+#### Making Updates
+
+Before making updates, be sure you're up to date with the latest `main` and have updated your dependencies.
```

Comment:
Do we have a different branches for prerelease? Should we notice about this branches too?

Reply:
Updated! Thank you!